ouster_client/include/ouster/client.h

```diff
@@ -213,5 +230,14 @@ int get_lidar_port(client& cli);
  */
 int get_imu_port(client& cli);
 
+/**
+ * Check if ip address in multicast range.
+ *
+ * @param[in] addr ip address to test.
+ *
+ * @return true if addr is in multicast range.
+ */
+bool in_multicast(const char* addr);
```

Comment:
We largely depend on `std::string` so I would rather use it than raw string
```suggestion
bool in_multicast(const std::string& addr);
```
Within the function body we can call addr.c_str()

Reply:
Fixed in [450aefc](https://github.com/ouster-lidar/ouster_example/pull/480/commits/450aefce39c22c60870b515ae91e9f55bf4446d1)